docs/cobbler.rst

```diff
@@ -930,10 +930,38 @@ Example:
 
     $ cobbler aclsetup
 
-Cobbler buildiso
+cobbler buildiso
 ================
 
-Example:
+All parameters are optional on the CLI. This command may not behave like you expect it without installing additional
+dependencies and configuration. The in depth explanation can be found at :ref:`building-isos`.
+
++--------------+-------------------------------------------------------------------------------------------------------+
+| Name         | Description                                                                                           |
++--------------+-------------------------------------------------------------------------------------------------------+
+| iso          | Output ISO to this file. If the file exists it will be truncated to zero before.                      |
++--------------+-------------------------------------------------------------------------------------------------------+
+| profiles     | Use these profiles only for collection information.                                                   |
++--------------+-------------------------------------------------------------------------------------------------------+
+| systems      | Use these systems only for collection information.                                                    |
```

Comment:
Not really sure if it's wrong, but `information collection` sounds better for me

Reply:
I think the word `collection` here means the Cobbler collections rather then the expression 'to collect information'. Otherwise `... for collecting information` sounds good, too.

Reply:
Should be fixed with d099022